docs/articles/documentation/test-api/actions/click.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+title: Click
+permalink: /documentation/test-api/actions/click.html
+---
+# Click
+
+Clicks a webpage element.
+
+```text
+click( selector [, options] )
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | -----------------------------------------------------------------------------------------------------------------------
+`selector`             | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the webpage element being clicked. See [Selecting Target Elements](index.md#selecting-target-elements).
+`options` *(optional)* | Object                                            | A set of options that provide additional parameters for the action. See [Click Action Options](action-options.md#click-action-options).
+
+The following example shows how to use the `click` action to tick a checkbox element.
+
+```js
+import { expect } from 'chai';
+
+test('Click a check box and check its state', async t => {
```

Comment:
All examples are missing fixture and page declaration. Are these tests can be run on example page or they are just synthetic examples?